addon/src/webextension/icons/icon-16.svg

```diff
@@ -0,0 +1 @@
+<svg width="16" height="14" viewBox="0 0 16 14" xmlns="http://www.w3.org/2000/svg"><path fill="context-fill" d="M14.962 6.169c.51.253.865.738.954 1.3.092.83-.476 1.884-1.51 2.85.48 1.481 1.027 3.681.031 3.681-.724 0-1.68-1.05-2.42-2.079a10.922 10.922 0 0 1-3.534 1.019 10.847 10.847 0 0 1-4.886-.59C2.916 13.223 2.123 14 1.5 14c-.832 0-.587-1.538-.204-2.908a3.074 3.074 0 0 1-1.26-1.874c-.124-1.132 1.05-2.069 2.94-2.747C3.07 3.95 4.372 0 8 0a4.365 4.365 0 0 1 3.867 2.35c1.2-.207 2.415-.324 3.633-.35.5 0 .5 0 .5.5a12.08 12.08 0 0 1-1.038 3.669zM8 1.45c-2.212 0-3 2.05-3.433 4.65C4.42 6.99 4 8 7 8c3.5 0 4.487-1.609 4.43-1.928C11 3.696 10.2 1.45 8 1.45z"/></svg>
```

Comment:
Hmm, not sure what happened... I restarted and now it works fine when I install via about:debugging. So, mainly just needs a version bump I think